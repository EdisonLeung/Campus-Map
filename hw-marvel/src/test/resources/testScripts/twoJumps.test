# Tests pathfinding from one node to another with 2 jumps

# Create graph
CreateGraph twoJumps

# Add nodes
AddNode twoJumps a
AddNode twoJumps b
AddNode twoJumps c

# Add Edges
AddEdge twoJumps a b AB
AddEdge twoJumps b c BC

# path finding
FindPath twoJumps a c