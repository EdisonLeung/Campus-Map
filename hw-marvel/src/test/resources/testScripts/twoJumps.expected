# Tests pathfinding from one node to another with 2 jumps

# Create graph
created graph twoJumps

# Add nodes
added node a to twoJumps
added node b to twoJumps
added node c to twoJumps

# Add Edges
added edge AB from a to b in twoJumps
added edge BC from b to c in twoJumps

# path finding
path from a to c:
a to b via AB
b to c via BC